server.js

```diff
@@ -0,0 +1,23 @@
+require('dotenv').config();
+const cors = require('cors');
+const express = require('express');
+const { logger, errorHandler } = require('./middleware');
+const { seed } = require('./utils');
+const app = express();
```

Comment:
Add a newline before this, just to visually separate the import section

```suggestion

const app = express();
```